webapp/src/Service/CheckConfigService.php

```diff
@@ -337,6 +346,36 @@ public function checkTmpdirWritable()
                  $tmpdir)];
     }
 
+    public function checkHashTime()
+    {
+        $tmpuser = new User();
+        $counter = 0;
+        $time_start = microtime(true);
+        do {
+            $plainPassword = 'ryanpass';
+            $encoded = $this->passwordEncoder->encodePassword($tmpuser, $plainPassword);
+            // $user->setPassword($encoded);
```

Comment:
This comment can be removed

Reply:
Thats something I wanted to ask you, I assume the encodePassword will already do the cost (so its not a promise to the DB?)

Reply:
Correct.